.circleci/ci.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+set -ex
+cd "$(dirname "$0")"
+
+TENSORFLOW_BUILD_TAG="${TENSORFLOW_BUILD_TAG:-centos}"
+TENSORFLOW_DOCKER_BUILD_TAG="${TENSORFLOW_DOCKER_BUILD_TAG:-latest}"
+BAZEL_OUTPUT_BASE="${BAZEL_OUTPUT_BASE:-/var/tmp/bazel-output-base}"
+BUILD_CONFIG="${BUILD_CONFIG:-mkl}"
+
+if [[ ! -z "${BAZEL_VERSION}" ]]
+then
+    DOCKER_BUILD_ARGS="${DOCKER_BUILD_ARGS} --build-arg BAZEL_VERSION=${BAZEL_VERSION}"
+fi
+
+mkdir -p "${BAZEL_OUTPUT_BASE}"
+mkdir -p build
+
+
+
+mkdir -p "../${OUTPUT_DIR}"
+
+if [[ -z ${TENSORFLOW_SKIP_DOCKER_BUILD} ]] ; then
+    docker build \
+        ${DOCKER_BUILD_ARGS} \
+        -t 794612149504.dkr.ecr.us-east-1.amazonaws.com/tensorflow-ci:${TENSORFLOW_DOCKER_BUILD_TAG} \
```

Comment:
* the docker image is never pushed to the registry. is that on purpose?
* let's rename the docker image from tensorflow-ci to centos-tensorflow-ci - I believe it should contain centos

Reply:
>the docker image is never pushed to the registry. is that on purpose?

We should push it, I didn't have time to add that yet